docs/dataformats/index.rst

```diff
@@ -39,7 +39,7 @@ Type              Format Name          Gammapy
 Events            EVENTS               `~gammapy.data.EventList`                         GEventList_
 ----------------- -------------------- ------------------------------------------------- ----------------------------
 Counts            3D                   `~gammapy.cube.SkyCube`                           `GCTAEventCube`_
-Counts            Image                `~gammapy.image.SkyMap`                           `GSkyMap`_
+Counts            Image                `~gammapy.image.SkyImage`                           `GSkyMap`_
```

Comment:
Fix indentation of the last column (or the Sphinx build will fail and the table won't be rendered correctly).